index.html

```diff
@@ -34,7 +21,24 @@
 		<!-- (Optional) Link to a 60x60 bookmark icon for iOS 
 		<link rel="apple-touch-icon-precomposed" href="../style/favicons/60x60/flag-red.png"/>
 		-->
-		
+
+    
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="css/skeleton.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
create an issue?

On Thu, Jul 30, 2015 at 11:02 AM, Ernie Hsiung <notifications@github.com>
wrote:

>> @@ -34,7 +21,24 @@
>>  		<!-- (Optional) Link to a 60x60 bookmark icon for iOS 
>>  		<link rel="apple-touch-icon-precomposed" href="../style/favicons/60x60/flag-red.png"/>
>>  		-->
>> -		
>> +
>> +    
>> +    <link rel="stylesheet" type="text/css" href="css/normalize.css" />
>> +    <link rel="stylesheet" type="text/css" href="css/skeleton.css" />
>> +    <link rel="stylesheet" type="text/css" href="css/style.css" />
>> +    
>> +    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
> So a thing to do is to for future use is to replace `src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"` with just `src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"`. This will prevent any JS errors (which would crash the wizard) if someone tried to access the URL with an `https` protocol.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/codeforamerica/mdc-step-wizard/pull/62/files#r35900120